dbus/dbus.go

```diff
@@ -148,6 +148,15 @@ func newConnection(createBus func() (*dbus.Conn, error)) (*Conn, error) {
 	return c, nil
 }
 
+// GetProperty returns values of a org.freedesktop.systemd1.Manager interface property
+func (c *Conn) GetProperty(prop string) (string, error) {
```

Comment:
what about GetManagerProperty  ? my understanding is that we can use it to fetch only systemd manager specific property, like Version, Features, Logtarget etc. Also, some of these returns non-string (array, int etc). Should i instead use interface{} as return type or something more appropriate ?